tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -188,4 +188,92 @@ partial class Build
                 CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
             }
         });
+
+    Target BuildAndRunProfilerLinuxIntegrationTests => _ => _
+        .Requires(() => !IsWin)
+        .Requires(() => IsLinux)
+        .Requires(() => !IsArm64)
+        .After(BuildTracerHome)
+        .After(BuildProfilerHome)
+        .After(BuildNativeLoader)
+        .After(ZipMonitoringHome)
+        .Description("Builds and runs the profiler linux integration tests.")
+        .DependsOn(CompileProfilerSamplesLinux)
+        .DependsOn(CompileProfilerLinuxIntegrationTests)
+        .DependsOn(RunProfilerLinuxIntegrationTests);
+
+    Target CompileProfilerLinuxIntegrationTests => _ => _
+        .Unlisted()
+        .After(PublishProfilerLinux)
+        .After(CompileProfilerSamplesLinux)
+        .Executes(() =>
+        {
+            // Build the actual integration test projects for x64
+            var integrationTestProjects = ProfilerDirectory.GlobFiles("test/*.IntegrationTests/*.csproj");
+            DotNetBuild(x => x
+                    // .EnableNoRestore()
+                    .EnableNoDependencies()
+                    .SetConfiguration(BuildConfiguration)
+                    .SetTargetPlatform(MSBuildTargetPlatform.x64)
+                    .SetNoWarnDotNetCore3()
+                    .When(TestAllPackageVersions, o => o.SetProperty("TestAllPackageVersions", "true"))
+                    .When(IncludeMinorPackageVersions, o => o.SetProperty("IncludeMinorPackageVersions", "true"))
+                    .When(!string.IsNullOrEmpty(NugetPackageDirectory), o =>
+                        o.SetPackageDirectory(NugetPackageDirectory))
+                    .CombineWith(integrationTestProjects, (c, project) => c
+                        .SetProjectFile(project)));
+        });
+
+
+    Target CompileProfilerSamplesLinux => _ => _
+        .Unlisted()
+        .Executes(() =>
+        {
+            var samplesToBuild = ProfilerSamplesSolution.GetProjects("*");
+
+            // Always x64
+            DotNetBuild(x => x
+                    .SetConfiguration(BuildConfiguration)
+                    .SetTargetPlatform(MSBuildTargetPlatform.x64)
+                    .SetNoWarnDotNetCore3()
+                    .When(!string.IsNullOrEmpty(NugetPackageDirectory), o => o.SetPackageDirectory(NugetPackageDirectory))
+                    .CombineWith(samplesToBuild, (c, project) => c
+                        .SetProjectFile(project)));
+        });
+
+    Target RunProfilerLinuxIntegrationTests => _ => _
+        .After(CompileProfilerSamplesLinux)
+        .After(CompileProfilerLinuxIntegrationTests)
+        .Description("Runs the profiler linux integration tests")
+        .Requires(() => !IsWin)
+        .Requires(() => !IsArm64)
```

Comment:
nit (I assume these don't work on OSX yet)
```suggestion
        .Requires(() => IsLinux && !IsArm64)
```